scripts/romanisim-make-stack

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python
+
+import sys
+import argparse
+import datetime
+import logging
+import csv
+from copy import deepcopy
+import asdf
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+from astropy.table import Table
+from astropy.io import fits
+from astropy.time import Time
+import galsim
+from romanisim import catalog, image, wcs, persistence
+from romanisim.parameters import default_parameters_dictionary
+from romanisim import log
+from romanisim import parameters
+
+NUMBER_OF_DETECTORS = 18
```

Comment:
Can we move this into romanisim.parameters?

Reply:
Yeah, it was on my TODO list during cleanup. Moved.